consensus/Cargo.toml

```diff
@@ -32,4 +32,7 @@ governor = { workspace = true }
 prost-build = { workspace = true }
 
 [dev-dependencies]
-tracing-subscriber = { workspace = true }
\ No newline at end of file
+tracing-subscriber = { workspace = true }
+
+[lib]
+bench = false
```

Comment:
Do you have any docs on where this comes from?

Reply:
Yeah I added it to the bottom part of the description of the PR :)

Reply:
my b 😓